hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -595,66 +607,36 @@ class Statistics(IntEnum):
     always = 2
 
 
-class Verbosity(object):
+@unique
+class Verbosity(IntEnum):
+    quiet = 0
+    normal = 1
+    verbose = 2
+    debug = 3
+
+    @staticmethod
+    def _get_default():
+        var = os.getenv('HYPOTHESIS_VERBOSITY_LEVEL')
+        if var is not None:  # pragma: no cover
+            note_deprecation(
+                'The $HYPOTHESIS_VERBOSITY_LEVEL environment variable is '
```

Comment:
Nit: I don't think the variable name should be prefixed with a dollar. As well as being a shell artifact, that refers to its value and not its name.